src/Grpc/JsonTranscoding/src/Microsoft.AspNetCore.Grpc.JsonTranscoding/Internal/ErrorMessageHelper.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.Grpc.JsonTranscoding.Internal;
+
+internal static class ErrorMessageHelper
+{
+    internal static string BuildErrorMessage(string message, Exception exception, bool? includeExceptionDetails)
+    {
+        if (includeExceptionDetails ?? false)
+        {
+            return message + " " + CommonGrpcProtocolHelpers.ConvertToRpcExceptionMessage(exception);
```

Comment:
What's wrong with `exception.ToString()`? Are we trying to avoid stack traces?

Reply:
Yes